code/game/objects/items/devices/multitool.dm

```diff
@@ -53,18 +53,60 @@
 				for(var/mob/camera/aiEye/A in chunk.seenby)
 					var/turf/eye_turf = get_turf(A)
 					if(get_dist(our_turf, eye_turf) < 8)
+						found_eye = 2
+						break
+					if(get_dist(our_turf, eye_turf) < 20) //AI is near you, be careful
 						found_eye = 1
 						break
 
-	if(found_eye)
+	if(found_eye == 1)
+		icon_state = "[initial(icon_state)]_yellow"
+	else if(found_eye == 2)

```

Comment:
I think at the beginning of the file you do like '## EYE_FOUND 2' or some bullshit then substitute 2 here for EYE_FOUND

For an actual example of the syntax I'd check human_defines.dm or something